modules/util/portal-tools-soy-builder/src/main/resources/META-INF/m2e/lifecycle-mapping-metadata.xml

```diff
@@ -5,6 +5,7 @@
 		<pluginExecution>
 			<pluginExecutionFilter>
 				<goals>
+					<goal>build-soy</goal>
```

Comment:
In other goals we just use the word `build` and not `build-soy`, do we need to update that here?